express-basic/list/utility.js

```diff
@@ -0,0 +1,33 @@
+function someFunc(colors){
+  return Object.keys(colors).reduce((list, key) => {
```

Comment:
there's no need for a separate function - move this code to line 31 and assign the output to the colors variable